docker/imagej/Dockerfile

```diff
@@ -25,40 +45,25 @@ ENV LIBRARY_PATH=/lib:/usr/lib
 RUN pip3 install cython --no-cache-dir && \
     pip3 install pyimagej==1.4.1 --no-cache-dir
 
-# Polus platform file extension
-ENV POLUS_IMG_EXT=".ome.tif"
-ENV POLUS_TAB_EXT=".csv"
+# Update Maven local repository path
+RUN mkdir -p /usr/share/maven/conf
+RUN echo '<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" ' \
+     'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ' \
+     'xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 ' \
+     'http://maven.apache.org/xsd/settings-1.0.0.xsd">' \
+     '<localRepository>/etc/m2</localRepository>' \
+     '</settings>' > /usr/share/maven/conf/settings.xml
 
-# Container log level
-ENV POLUS_LOG="INFO"
+# Ensure the /etc/m2 directory exists
+RUN mkdir -p /etc/m2
 
-# Data and code directories
-ENV EXEC_DIR="/opt/executables"
-ENV DATA_DIR="/data"
+# Set the correct permissions
+RUN chmod -R 777 ${EXEC_DIR} && \
+    chmod -R 777 ${DATA_DIR} && \
+    chmod -R 777 /etc/m2
 
-RUN mkdir ${EXEC_DIR} && \
-    mkdir ${DATA_DIR}
-
-# create a non-root user named polusai
-RUN groupadd polusai && \
-    useradd --no-log-init -r -m -g polusai polusai && \
-    usermod -aG sudo polusai
-
-ENV HOME=/home/polusai
-RUN chmod -R o+rx ${HOME}
-# NOTE: By default, CWL adds --user=... to the docker run commands.
-# https://github.com/common-workflow-language/cwltool/blob/7b8f174828e3c07b3067e4f88568124f7c878d2d/cwltool/docker.py#L362-L366
-# Since we do not control the userid and/or groupid on the
-# host machine(s) where this docker image will be executed,
-# we must make HOME readable and executable for all users! Otherwise:
-# /usr/bin/python3: Error while finding module specification for 'polus.plugins...'
-# (ModuleNotFoundError: No module named 'polus')
-
-RUN chown -R polusai ${EXEC_DIR} && \
-    chown -R polusai ${DATA_DIR} && \
-    chmod -R u+rwx ${EXEC_DIR} && \
-    chmod -R u+rwx ${DATA_DIR}
-
-USER polusai
 RUN python3 -c 'import bfio; bfio.start()'
 RUN python3 -c 'import imagej; ij = imagej.init(); print(ij.getVersion())'
+
+# Set the correct permissions again, for sanity
+RUN chmod -R 777 /etc/m2
```

Comment:
I am not clear why you have to set the permissions again? (similar in other dockerfiles)

Reply:
once we run line 67, new files get added to `/etc/m2`. I just want to be absolutely sure that all the new files can be read, and executed by any user.